docs/_includes/expressive-loader.html

```diff
@@ -0,0 +1,584 @@
+            <div class="expressive-loader__line expressive-loader__line-3" style="width: 18%;"></div>
+            <div class="expressive-loader__line expressive-loader__line-4" style="width: 26%;"></div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  {% highlight html %}
+  <div class="expressive-loader">
+    <div class="expressive-loader__message-container">
+      <div aria-hidden="true" class="expressive-loader__message">Hang tight.</div>
+      <div role="status" id="expressive-loader-message-1" class="expressive-loader__message expressive-loader__message--initial"></div>
+    </div>
+    <div role="progressbar" aria-label="Loading..." aria-describedby="expressive-loader-message-1" class="expressive-loader__progress">
+      <div class="expressive-loader__lines">
+        <div class="expressive-loader__line expressive-loader__line-1" style="width: 13%;"></div>
+        <div class="expressive-loader__line expressive-loader__line-2" style="width: 33%;"></div>
+        <div class="expressive-loader__line expressive-loader__line-3" style="width: 18%;"></div>
+        <div class="expressive-loader__line expressive-loader__line-4" style="width: 26%;"></div>
+      </div>
+    </div>
+  </div>
+  {% endhighlight %}
+
+  <h4>
+    Message animation
+  </h3>
+  <p>
+    If there are multiple messages to display, cycle them in and out of view using Javascript.
+    To cycle a message into view:
+    <ol>
+      <li>Animate in the message:
+        <ol>
+          <li>Put the message content in the aria-hidden element with class <code>expressive-loader__message</code>. This element is used just for the visual message animation; it is not read to screen readers.</li>
+          <li>Add the <code>expressive-loader__message--in</code> class to animate that message into view.</li>
+        </ol>
+      </li>
+      <li>
+        After the message animates in:
+        <ol>
+          <li>
+            Put the message content in the status element with class <code>expressive-loader__message</code>. This will read the message aloud to screen reader users.
+          </li>
+          <li>
+            Remove the <code>expressive-loader__message--in</code> class from the aria-hidden element.
+          </li>
+        </ol>
+      </li>
+      <li>
+        After a fixed amount of time, add the <code>expressive_loader__message--out</code> class
+        to the current message to animate it out of view. This can happen concurrently with the next message animating in.
+      </li>
+    </ol>
+  </p>
+
+  <div class="demo">
+    <div class="demo__inner">
+      <div class="expressive-loader">
+        <div class="expressive-loader__message-container">
+          <div aria-hidden="true" class="expressive-loader__message">
+            We're finishing your order.
+          </div>
+          <div role="status" id="expressive-loader-message-2" class="expressive-loader__message">
```

Comment:
I don't know if it's still an issue, but I know VO used to have trouble detecting live regions changes if the content wasn't inside of a subnode.

I would always have to change from something like this:

```
<div role="status" id="expressive-loader-message-2" class="expressive-loader__message">
   Hang tight.
</div>
```

To this:

```
<div role="status" id="expressive-loader-message-2" class="expressive-loader__message">
   <span>Hang tight.</span>
</div>
```

Again, it could no longer be an issue, but worth a quick test.

Reply:
Tested with VO on Chrome & Safari and it's reading well.